api/python/provisioner/utils.py

```diff
@@ -304,10 +304,10 @@ def run_subprocess_cmd(cmd, **kwargs):
     )
 
     _kwargs.update(kwargs)
-    _kwargs['check'] = True
 
     if type(cmd) is str:
         cmd = cmd.split()
+        _kwargs['check'] = True
 
```

Comment:
`check=True` did not fetch the right output for some shell commands earlier addressed with `Popen`, so moved it to other non-list commands.

Reply:
I don't think that we can relate command form (str or list) with check. Please don't mix that. If you need to be able to pass `check=False` then just make it as argument opposite to current force approach, e.g.:
```python
def run_subprocess_cmd(cmd, check=True, **kwargs):
    _kwargs = dict(
        universal_newlines=True,
        stdout=subprocess.PIPE,
        stder=subprocess.PIPE,
        check=check
    )
...
```

Reply:
@andkononykhin no, what I meant was, when `check=True` and some shell commands return non-zero code, it raises exception instead of returning empty stdout.
Addressed now, thank you!